bin/GameIndex.dbf

```diff
@@ -2244,6 +2254,8 @@ Serial = SCES-53409
 Name   = Sly 3
 Region = PAL-M11
 Compat = 5
+vuRoundMode = 0 // Fix issues with camera in some levels.
+vuClampMode = 2 // Fix game engine issues like the bombs not exploding and the inability to take a ladder throughout the game.
```

Comment:
`Fixes not being able to climb a ladder and fixes Bentley's bombs not working in the Episode 5 mission "Dynamic Duo".`

I assume you meant climbing a ladder, not literally taking a ladder.